showcase3D.py

```diff
@@ -14,10 +14,10 @@
 colors = ((1, 0.8, 0), (0, 0.5, 0), (0.95, 0.95, 0.95), (0, 0, 0.75), (0.9, 0.5, 0), (0.7, 0, 0))
 moves = {
```

Comment:
Maybe put these variables in uppercase to show that they are constants

Reply:
Okay ✅